prisma-fmt/src/text_document_completion/datasource.rs

```diff
@@ -0,0 +1,172 @@
+use std::collections::HashMap;
+
+use lsp_types::{
+    CompletionItem, CompletionItemKind, CompletionList, Documentation, InsertTextFormat, MarkupContent, MarkupKind,
+};
+
+use super::generate_pretty_doc;
+
+pub(super) fn schemas_completion(completion_list: &mut CompletionList) {
+    completion_list.items.push(CompletionItem {
+        label: "engines schemas".to_owned(),
+        insert_text: Some(r#"schemas = [$0]"#.to_owned()),
+        insert_text_format: Some(InsertTextFormat::SNIPPET),
+        kind: Some(CompletionItemKind::FIELD),
+        documentation: Some(Documentation::MarkupContent(MarkupContent {
+            kind: MarkupKind::Markdown,
+            value: generate_pretty_doc(
+                r#"schemas = ["foo", "bar", "baz"]"#,
+                "The list of database schemas. [Learn More](https://pris.ly/d/multi-schema-configuration)",
+                None,
+            ),
+        })),
+        // detail: Some("schemas".to_owned()),
+        ..Default::default()
+    });
+}
+
+pub(super) fn relation_mode_completion(completion_list: &mut CompletionList) {
+    completion_list.items.push(CompletionItem {
+        label: "engines relationMode".to_owned(),
+        insert_text: Some(r#"relationmode = $0"#.to_owned()),
```

Comment:
I typically tend to prefer not having the `" "` as it's then easier for someone to just add the quote and that trigger the completion rather than necessarily know about the manual completion keybind

Reply:
👍🏾